src/components/tableOfContents/js/TableOfContents.js

```diff
@@ -20,25 +20,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     *******/
     fluid.registerNamespace("fluid.tableOfContents");
 
-
-    fluid.tableOfContents.insertAnchor = function (name, element, anchorClass) {
-       // In order to resolve FLUID-4453, we need to make sure that the owner document is correctly
-       // taken from the target element (the preview may be in an iframe)
-        var anchor = $("<a></a>", element.ownerDocument);
```

Comment:
@simonbates I believe it was only an issue because of the injection, but you could test it out by running the full page manual-tests for this branch ( e.g http://build.fluidproject.org/infusion/tests/manual-tests/framework/preferences/fullPage-schema/index-nativeHTML.html ). 

Also, looking at the [FLUID-4453](https://issues.fluidproject.org/browse/FLUID-4453), and we've also mostly removed the jQuery requirements in the iframes as well.